Makefile

```diff
@@ -32,6 +32,18 @@ test-integration:
 test-integration-parallel:
 	go run ./tests/integration/... --include-multi-consumer --parallel
 
+# run full integration tests in sequence (including multiconsumer) using latest tagged gaia
+test-gaia-integration:
```

Comment:
Instructions for running these should be added to `testing.md`

Reply:
Added!